### PR TITLE
Defaults

### PR DESCRIPTION
**Pull request by steveburton4**
Change corporate storage pipeline to use batch jobs and create production jobs for storage and manifests. Create ones that can be running partition at a time if needed but automate the full ones.

### CODE REVIEW

File: ci/utility/corporate-storage-coalescer/jobs/manifests-production-equalities-per-partition.yml [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+            params:
+              TF_WORKSPACE: default
+        - .: (( inject meta-corporate-storage-coalescer.plan.terraform-output-internal-compute ))
+          config:
+            params:
+              TF_WORKSPACE: default
+        - .: (( inject meta-corporate-storage-coalescer.plan.terraform-output-ingest-consumers ))
+          config:
+            params:
+              TF_WORKSPACE: default
```
Comment (connoravo) [inline ci/utility/corporate-storage-coalescer/jobs/manifests-production-equalities-per-partition.yml]: TF workspace default in a production job

File: ci/utility/corporate-storage-coalescer/jobs/manifests-production-all.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+              TF_WORKSPACE: default
+        - .: (( inject meta-corporate-storage-coalescer.plan.terraform-output-internal-compute ))
+          config:
+            params:
+              TF_WORKSPACE: default
+        - .: (( inject meta-corporate-storage-coalescer.plan.terraform-output-ingest-consumers ))
+          config:
+            params:
+              TF_WORKSPACE: default
+
```
Comment (connoravo) [inline ci/utility/corporate-storage-coalescer/jobs/manifests-production-all.yml]: TF workspace default in a production job

File: ci/utility/corporate-storage-coalescer/jobs/manifests-production-equalities-per-partition.yml (hunk 1 above)
Comment (steveburton4) [reply ci/utility/corporate-storage-coalescer/jobs/manifests-production-equalities-per-partition.yml]: Fixed

File: ci/utility/corporate-storage-coalescer/jobs/manifests-production-all.yml (hunk 2 above)
Comment (steveburton4) [reply ci/utility/corporate-storage-coalescer/jobs/manifests-production-all.yml]: Fixed

**Review by connoravo — APPROVED** [review]

**Pull request merged**